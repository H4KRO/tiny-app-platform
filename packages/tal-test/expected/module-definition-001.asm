main()
  entry:
    FunctionRef       name: add_0
    DeclareLocal      name: add, mutable: false, hasInitialValue: true
    Pop               inBlock: false
    Literal "add"
    Local             name: add
    Literal 1
    MakeRecord
add_0([object Object], [object Object])
  entry:
    Local             name: a
    Local             name: b
    Intrinsic         operation: INTRINSIC_ADD
    MakeArrayForBlock count: 1
