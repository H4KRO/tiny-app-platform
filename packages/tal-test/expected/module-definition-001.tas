fun add(a, b) { a + b }

{add: add}
