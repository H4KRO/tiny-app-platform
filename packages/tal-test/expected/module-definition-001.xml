<program>
  <declarelocal>
    <function>
      <block>
        <binaryoperator>
          <local />
          <local />
        </binaryoperator>
      </block>
    </function>
  </declarelocal>
  <record>
    <recordentry>
      <local />
    </recordentry>
  </record>
</program>
